test/TestAssets/Directory.Build.props

```diff
@@ -1,6 +1,10 @@
+<!-- Prevent test asset projects from picking up the repo's root Directory.Build.props. -->
 <Project>
-  <!-- A copied Directory.Build.props file to prevent test asset projects from picking up the repo's Directory.Build.props' -->
+
   <PropertyGroup>
     <ManagePackageVersionsCentrally>false</ManagePackageVersionsCentrally>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```

Comment:
I wasn't aware this property existed. That's pretty cool.

Reply:
Same comment as above, already there: https://github.com/dotnet/sdk/blob/main/src/Assets/Directory.Build.props